tests/automation/types/testing.ts

```diff
@@ -11,10 +13,52 @@ export type Group = {
   userThree: User;
 };
 
-export type Strategy = 'data-testid' | 'class' | ':has-text';
+export type ConversationType = '1:1' | 'group' | 'community' | 'note-to-self';
+
+export type DMTimeOption =
+  | 'input-10-seconds'
+  | 'time-option-10-seconds'
+  | 'time-option-30-seconds'
+  | 'time-option-1-minute'
+  | 'time-option-5-minutes'
+  | 'time-option-1-hour'
+  | 'time-option-12-hours'
+  | 'time-option-1-day'
+  | 'time-option-1-week'
+  | 'time-option-2-weeks';
+
+export type DisappearOpts1o1 = [
+  '1:1',
+  'disappear-after-read-option' | 'disappear-after-send-option',
+  DMTimeOption,
+];
+
+export type DisappearOptsGroup = [
+  'group' | 'note-to-self',
+  'disappear-after-send-option',
+  DMTimeOption,
+];
+
+export type MergedOptions = DisappearOpts1o1 | DisappearOptsGroup;
+
+export type StrategyExtractionObj =
+  | {
+      strategy: Extract<Strategy, ':has-text' | 'class'>;
+      selector: string;
+    }
+  | {
+      strategy: Extract<Strategy, 'data-testid'>;
+      selector: DataTestId;
+    };
+
+export type WithPage = { window: Page };
+export type WithMaxWait = { maxWait?: number };
+export type WithRightButton = { rightButton?: boolean };
```

Comment:
What exactly do you mean?

Reply:
What I mean is that many other actions need those 2
![image](https://github.com/oxen-io/session-playwright/assets/1544279/29873d28-87a6-474f-b556-87a56ca36d45)
![image](https://github.com/oxen-io/session-playwright/assets/1544279/ff651e80-acf0-4549-a6b5-36bd3b4dd1bc)

 it would make sense to use the same syntax  as here to make this all cleaner 
![image](https://github.com/oxen-io/session-playwright/assets/1544279/8caf4ec3-dc31-415e-a4cd-58581dfe04c9)